eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -56,4 +56,8 @@ eventMesh.server.registry.fetchRegistryAddrIntervalInMills=20000
 #eventMesh.server.defibus.client.comsumeTimeoutInMin=5
 
 #connector plugin
-eventMesh.connector.plugin.type=rocketmq
\ No newline at end of file
+eventMesh.connector.plugin.type=rocketmq
+
+#acl plugin
+eventMesh.server.acl.enabled=false
+eventMesh.acl.plugin.type=acl
```

Comment:
eventMesh.server.security.enabled & eventMesh.security.plugin.type maybe better

Reply:
OK， I will adjust it.